.msggen.json

```diff
@@ -2874,7 +2874,7 @@
             "deprecated": null
         },
         "Getinfo.address[]": {
-            "added": "pre-v0.10.1",
+            "added": "v23.02",
```

Comment:
this is correct? I think I miss the meaning of the `.msgen.json` so this question can be stupid

Reply:
It's the version when the field went from being optional to mandatory, hence we need to mark it as new, so the inference for optionality detects it as something that used to be optional and now is mandatory.

Yes, the semantics of `added` are not well-defined yet, and a `required_since` field would be clearer, but that seems like absolute overkill when all we care about is whether we need to downgrade a mandatory field to be optional because it used to be optional before. Seems to me like a very rare case.

Reply:
yeah, I was thinking that it had a different meaning :) all good then